algorithms/class1.html

```diff
@@ -187,7 +188,7 @@ <h2>Goal of Algorithms</h2>
                 <h2>Exercise time!</h2>
                 <h3 class='highlight'>What is an algorithm you use?</h3>
                 <br>
-                <p>Write down steps for several versions of an algorithm to solve an everyday problem</p>
+                <p>Find a partner and pick a problem. Come up with multiple versions of an algorithm and write them down.</p>
```

Comment:
Might want to expand on what 'pick a problem' is. Perhaps everyday situation or something a bit more specific

Reply:
There are examples just below: https://github.com/gdiseattle/computer-science-fundamentals/pull/2/files#diff-86d3c912d2d3b2efc183261800c99d00R196